network-api/networkapi/wagtailpages/templates/about/about_nav.html

```diff
@@ -1,22 +1,12 @@
 {% load bg_nav_tags i18n wagtailroutablepage_tags %}
 
 {% with request.get_full_path as current_url %}
-{% if page %}
-  {% routablepageurl page 'about-why-view' as about_why_url %}
-  {% routablepageurl page 'how-to-use-view' as how_to_use_url %}
-  {% routablepageurl page 'min-security-view' as minimum_security_url %}
-  {% routablepageurl page 'methodology-view' as methodology_url %}
-  {% routablepageurl page 'contact-view' as contact_url %}
-  {% routablepageurl page 'press-view' as press_url %}
-{% else %}
-  {# TODO: Remove the below URLs when cleaning up old PNI code #}
-  {% url 'about-why-view' as about_why_url %}
-  {% url 'how-to-use-view' as how_to_use_url %}
-  {% url 'min-security-view' as minimum_security_url %}
-  {% url 'methodology-view' as methodology_url %}
-  {% url 'contact-view' as contact_url %}
-  {% url 'press-view' as press_url %}
-{% endif %}
+{% routablepageurl page 'about-why-view' as about_why_url %}
+{% routablepageurl page 'how-to-use-view' as how_to_use_url %}
+{% routablepageurl page 'min-security-view' as minimum_security_url %}
+{% routablepageurl page 'methodology-view' as methodology_url %}
+{% routablepageurl page 'contact-view' as contact_url %}
+{% routablepageurl page 'press-view' as press_url %}
```

Comment:
now I wish there was a routing block, so we don't have to repeated `routablepageurl page` so many times. Like:
```
{% routablepage with page=page, about_why_url='about-why-view', minimum_security_url='min-security-view', ... %}
```